doc/bazel/README.md

```diff
@@ -152,6 +158,44 @@ cc_binary(
 * **For C++**: Use `@iceoryx2//:iceoryx2-cxx-static` or
   `@iceoryx2//:iceoryx2-cxx-shared`
 
+### Feature Flags
+
+iceoryx2 provides several feature flags that can be configured using bazel
+build options.These flags allow you to enable or disable specific features
```

Comment:
```suggestion
build options. These flags allow you to enable or disable specific features
```

Reply:
Also some other issues the linter would catch. Did you install an auto-formatter?

Reply:
Is it part of a git hook?

Reply:
@elBoberido No but am not so sure if it is a good idea for that. Ideally the auto-corrections should be verified quickly (either with a git diff or in-editor with an auto formatter). There is to to the fixing from the terminal: https://github.com/eclipse-iceoryx/iceoryx2/blob/main/internal/scripts/check_markdown_linting.sh

Reply:
@orecham I know that script but forgot to run it. I'll create a git hook to run it on all markdown files in the commit if the markdown linter is available. That should be fairly fast and catches the issues immediately for regular contributors.

Reply:
@elBoberido If you want to do that, the script should already take care of finding the files on the current branch (including those not committed), so the hook would only need to run the script.

The hesitation I have is the time it takes (could be too long for a hook) and the line length correction which can mess up some intentional formatting.

Feel free to change the script however you want though.

Reply:
The hook would only do the check, not perform any changes. That would be up to the developer to trigger.